unstructured_inference/inference/layout.py

```diff
@@ -190,7 +189,7 @@ def __init__(
         self.number = number
         self.detection_model = detection_model
         self.element_extraction_model = element_extraction_model
-        self.elements: Collection[Union[LayoutElement, LocationlessLayoutElement]] = []
+        self.elements: Collection[LayoutElement] = []
```

Comment:
maybe worth document the breaking change here?

Reply:
Added a comment in [16b3437](https://github.com/Unstructured-IO/unstructured-inference/pull/232/commits/16b343729ca1bdafbd373789314bd023b91eb897).